app/src/zcashtestnet/res/values/strings.xml

```diff
@@ -1,4 +0,0 @@
-<resources>
```

Comment:
This deletion is not necessary for this fix, but I noticed we had this old way to configure testnet/mainnet which isn't being used anymore.